poker-hand-evaluator/src/evaluator/Hand.java

```diff
@@ -161,33 +158,28 @@ private boolean isRoyalFlush() {
 
     @Override
     public String toString() {
-        return "Hand{" +
-            "cards=" + cards +
-            ", handValue=" + handValue +
-            ", cardFrequencies=" + cardFrequencies + '}';
+        return "\nHand{" +
+                "cards=" + cards +
+                ", handValue=" + handValue +
+                ", cardFrequencies=" + cardFrequencies + '}';
     }
 
     @Override
     public int compareTo(Hand other) {
-        //if the handvalue ordinals are different, return with the hand with the higher ordinal
+
         if (this.handValue.ordinal() != other.handValue.ordinal()) {
-            System.out.printf("hand 1: %s hand 2: %s%n", this.handValue, other.handValue);
-            System.out.printf("compareto result: %s%n", this.handValue.compareTo(other.handValue));
+            //System.out.printf("\nhand 1: %s hand 2: %s%n", this.handValue, other.handValue);
+            //System.out.printf("\ncompareto result: %s%n", this.handValue.compareTo(other.handValue));
```

Comment:
please always remove dead code from production code - this comment is not necessary for understanding the code